requirements.txt

```diff
@@ -7,4 +7,5 @@ cloudshell-rest-api>=8.2.3.1,<9
 colorama
 giturlparse.py
 ruamel.yaml
-cryptography
\ No newline at end of file
+cryptography
+setuptools; python_version >= '3.12'
```

Comment:
why add the python_version requires?
we can also support older python versions

Reply:
I had a feeling, that since we need this only on python versions which doesn't have _setuptools_ built in, it will be safer to do it like that. So we won't break deps on other python versions.

Reply:
ok. looks good, am confirming